api/src/mailbox.rs

```diff
@@ -188,6 +188,7 @@ impl CommandId {
     // OCP LOCK Commands
     pub const OCP_LOCK_REPORT_HEK_METADATA: Self = Self(0x5248_4D54); // "RHMT"
     pub const OCP_LOCK_GET_ALGORITHMS: Self = Self(0x4741_4C47); // "GALG"
+    pub const OCP_LOCK_INITIALIZE_MEK_SECRET: Self = Self(0x494D_0B53); // "IKMS"
```

Comment:
```suggestion
    pub const OCP_LOCK_INITIALIZE_MEK_SECRET: Self = Self(0x494D_4B53); // "IMKS"
```

Looks like we also need to fix the mailbox code in https://github.com/chipsalliance/Caliptra/blob/6115e27ff9e75fbdb5b9ed2e05897b4195394ed7/doc/ocp_lock/lock_spec.ocp#L1652, which uses 0B instead of 4B for whatever reason.

Reply:
Ah thanks ! I opened https://github.com/chipsalliance/Caliptra/pull/589 to fix the spec.